pom.xml

```diff
@@ -101,6 +101,12 @@
 			<groupId>com.oracle.database.jdbc</groupId>
 			<artifactId>ojdbc11</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
You can replace this core dependency with the following and achieve the same result:

```
<dependency>
      <groupId>org.springframework.cloud</groupId>
      <artifactId>spring-cloud-starter-vault-config</artifactId>
</dependency>
```

It's always better in our case to rely on the boot starter dependencies that are controlled by the boot parent entry instead of going down and using the actual dependency.